lib/rules/wrap-iife.js

```diff
@@ -46,64 +58,95 @@ module.exports = {
                 nextToken && nextToken.value === ")";
         }
 
-        return {
+        /**
+        * Get the function node from an IIFE
+        * @param {ASTNode} node to evaluate
```

Comment:
This seems silly to me, but okay. I'm sure I just don't understanding something about JSDoc